project-maintainers.csv

```diff
@@ -624,21 +624,28 @@ Incubating,Contour,Steve Kriss,VMware,skriss,https://github.com/projectcontour/c
 ,,Tero Saarni,Ericsson,tsaarni,
 ,,Sunjay Bhatia,VMware,sunjayBhatia,
 ,,Nigel Brown,VMware,pnbrown,
-Incubating,Operator Framework: SDK,Eric Stroczynski,Red Hat,estroz,https://github.com/operator-framework/operator-sdk/blob/master/.cncf-maintainers
-,,Joe Lanford,Red Hat,joelanford,
+Incubating,Operator Framework: SDK,Joe Lanford,Red Hat,joelanford,,https://github.com/operator-framework/operator-sdk/blob/master/.cncf-maintainers
 ,,Ish Shah,Red Hat,theishshah,
-,,Camila Macedo,Red Hat,camilamacedo86,
-,,Jesus Rodriguez,Red Hat,jmrodri,
-,,Austin Macdonald ,Red Hat,asmacdo,

```

Comment:
As far as I know, these contributors are no longer active in the Operator SDK community, but I'd like to double check as I am more involved in the OLM side of the project. 

CC @estroz, @camilamacedo86, @jmrodri, @asmacdo

Reply:
Yes we are not